global.json

```diff
@@ -0,0 +1,6 @@
+{
+  "projects": [ "source" ],
```

Comment:
Apparently u don't need it, see https://github.com/dotnet/docs/pull/996#issuecomment-285505275